tuning/autotune.py

```diff
@@ -987,33 +987,65 @@ def parse_grouped_benchmark_results(
 ) -> Optional[list[str]]:
     """Update candidate_trackers and collect strings"""
     dump_list = []
+    incomplete_list: [
+        tuple[int, int]
+    ] = []  # format: [(candidate_id, device_id)], baseline will have candidate_id=0
 
     for same_device_results in grouped_benchmark_results:
+        dump_unsort_list: list[tuple[int, str]] = []
         for unet_candidate_result in same_device_results:
+            # skip if benchmark failed
```

Comment:
```suggestion
            # Skip if benchmark failed.
```

Let's try to follow the llvm coding standards here: https://llvm.org/docs/CodingStandards.html#commenting